src/schoology-extractor/.vscode/settings.json

```
@@ -103,5 +103,5 @@
         "pytest"
     ],
     "jupyter.jupyterServerType": "local",
-    "python.pythonPath": ".venv\\Scripts\\python.exe"
+    "python.pythonPath": "c:\\work\\fizz\\Ed-Fi-X-Fizz\\src\\schoology-extractor\\.venv\\Scripts\\python.exe"
```

```suggestion
    "python.pythonPath": ".venv\\Scripts\\python.exe"
```
My VS Code never changes this, wonder why yours does?